testsuite/tests/properties/entity_bind/expected_concrete_syntax.lkt

```diff
@@ -6,16 +6,12 @@ grammar foo_grammar {
 
 @abstract class FooNode implements Node {
 
-    @export fun prop (): Int
-
-    fun prop (): Int
+    @export @abstract @abstract_stub fun prop (): Int
```

Comment:
I think `@abstract_stub` or `@runtime_error` should replace `@abstracŧ` here ... Is that a mistake ?

Reply:
No, that was intended, but I’m okay with using `@not_implemented` or `@runtime_error` to mean both, with a slight preference towards `@not_implemented`.

Reply:
Ok, let's go with `@not_implemented` then!

Reply:
Done thanks